documentation/guide-blazegraph.asciidoc

```diff
@@ -0,0 +1,13 @@
+:toc: macro
```

Comment:
My intention was not to duplicate the database guides to devon4j as they have intentionally been moved away from devon4j to the devonfw guide before in order to be available for entire devonfw cross-cutting (not Java specific).
With manually duplicating the content we will end up in a maintenance problem since changes then have to be done in both places what will surely not work out.
Actually the link you replaced in the _Sidebar is working perfectly correct. So I do not get the problem.
Seems I have to study the issue #269 or did you actually get what link exactly is broken?

Reply:
I feel the link : https://github.com/devonfw/devonfw-guide/blob/master/general/db/guide-database is not taking the **.asciidoc** and the complete url should be : https://github.com/devonfw/devonfw-guide/blob/master/general/db/guide-database.asciidoc.

Reply:
Yep. Indeed the URLs (and also `link:` references) need to be complete including `.asciidoc` extension. Only in wikis the `link` references exclude the `.asciidoc` extension, but the wiki content is to be autogenerated by our CI pipelines and never maintained manually in `devonfw`.